subprojects/gradle-guides-plugin/src/main/java/org/gradle/docs/samples/internal/tasks/GenerateSanityCheckNegativeTests.java

```diff
@@ -0,0 +1,29 @@
+package org.gradle.docs.samples.internal.tasks;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.file.RegularFileProperty;
+import org.gradle.api.tasks.OutputFile;
+import org.gradle.api.tasks.TaskAction;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public abstract class GenerateSanityCheckNegativeTests extends DefaultTask {
```

Comment:
is this used anywhere yet?

Reply:
No, it's a leftover as it seemed cleaner to use explicit sanity checks.